src/main/java/uk/gov/justice/laa/crime/cfecrime/cma/enums/FullAssessmentResult.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.laa.crime.cfecrime.cma.enums;
```

Comment:
I think we need a comment indicating where these have been coped from - essentially these files must not be modified on a whim..

Reply:
commented.